environment.wake

```diff
@@ -125,7 +125,10 @@ publish runner =
     | findFail
   match jsonsResult
     Fail e =
-      def _ = printlnLevel logError "Failed to publish environment JSON runner: {e.getErrorCause}"
+      def _ = printlnLevel logError "environment-example-sifive: failed to publish environment JSON runner: {e.getErrorCause}"
```

Comment:
I like the prefixing idea you have here. Wonder if it'd be worth augmenting the logging functions with logger objects where you can specify a logger name that serves a similar function to your prefix, similar to [Java](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html#getLogger(java.lang.String)) and [Python](https://docs.python.org/3/library/logging.html#logging.getLogger).

Reply:
https://github.com/sifive/wake/issues/351